src/modals/collectible-modal/collectible-modal.tsx

```diff
@@ -146,15 +146,17 @@ export const CollectibleModal = () => {
   }, [fa.logo]);
 
   const handleShare = useCallback(async () => {
+    const urlEncodedData = encodeURIComponent(JSON.stringify({ ...collectible, description: '', holders: undefined }));
```

Comment:
why `description` and `holders` hardcode?

Reply:
we need to reduce the amount of data we send with a link, because of limits. Description is loading from the `useCollectibleInfo` and holders key isnt used on this screen, but they often contain a lot of data, thats why i choose them.